runtime/bcverify/staticverify.c

```diff
@@ -1894,6 +1908,12 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 
 	goto _leaveProc;			/* All is well */
 
+  _classNameError:
+	Trc_STV_j9bcv_verifyClassStructure_BadClassNameError(J9NLS_CFR_ERR_CP_BAD_CLASS_NAME__ID, classNameErrNo, cpIndex, className->slot1, className->bytes);
```

Comment:
What if `className` is never assigned?  Won't this crash dereferencing the null pointer?

Reply:
The className is initialized with NULL and assigned in the case of `CFR_CONSTANT_Class` in `j9bcv_verifyClassStructure`. 
```
		case CFR_CONSTANT_Class:
			/* Must be a UTF8. */
			utf8 = &classfile->constantPool[info->slot1];
			className = utf8;
```
 So It jumps to `_classNameError ` if anything wrong happens to `className` (assigned by `utf8`), which means if `className` is never assigned with `utf8`, then there is no problem to the class name in the constant pool. As a result, there is no way to reach `_classNameError ` and the tracepoint here.

Considering that any change in the future might unintentionally screws things up by dereferencing a null pointer in the tracepoint,  I agree to do the initialization first with `J9CfrConstantPoolInfo classNameInfo = {0, 0, 0, 0, 0, NULL, 0}" (all code there was updated accordingly), in which case it will end up with an empty class name if the class name is literally NULL in the constant pool (which in theory should never happen / will crash bcvCheckClassName(utf8) prior to `_classNameError `.